e2e/cypress/integration/enterprise/system_console/compliance/download_compliance_export_file_spec.js

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @system_console
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+describe('Compliance Export', () => {
+    before(() => {
+        cy.apiRequireLicenseForFeature('Compliance');
+        cy.apiUpdateConfig({
+            MessageExportSettings: {
+                DownloadExportResults: true,
+            },
+        });
+    });
+
+    // it('MM-T3435 - Download Compliance Export Files - CSV Format', () => {
+    //     // # Go to compliance page and enable export
+    //     gotoCompliancePage();
+    //     enableComplianceExport();
+
+    //     // # Navigate to a team and post an attachment
+    //     gotoTeamAndPostImage();
+
+    //     // # Goto compliance page and start export
+    //     gotoCompliancePage();
+    //     exportCompliance();
+
+    //     // # Get the download link
+    //     cy.get('@firstRow').findByText('Download').parents('a').should('exist').then((fileAttachment) => {
+    //         const fileURL = fileAttachment.attr('href');
+
+    //         // # Download the file
+    //         downloadAttachmentAndVerifyItsProperties(fileURL);
+    //     });
+    // });
+
+    // it('MM-T3438 - Download Compliance Export Files when 0 messages exported', () => {
```

Comment:
Same as above

Reply:
This is awkward 🤦🏻‍♂️😅 thanks @josephbaylon